store/search.go

```diff
@@ -15,68 +16,41 @@ import (
 var (
 	filterScriptBasePredicate       = `doc.containsKey(%q) == false || doc[%q].size() == 0`
 	filterScriptMatchValuePredicate = `doc[%q].value == %q`
+	defaultMaxResults               = 200
+	defaultMinScore                 = 0.01
+	defaultCachedTypesMapDuration   = 300
```

Comment:
Suggestion to put the cache into the config as an option. This way the user can flexibly adjust to their need.